docs/book/src/managed/adopting-clusters.md

```diff
@@ -15,10 +14,10 @@ and AzureASOManagedMachinePools. The [`asoctl import
 azure-resource`](https://azure.github.io/azure-service-operator/tools/asoctl/#import-azure-resource) command
 can help generate the required YAML.
 
-Caveats:
-- The `asoctl import azure-resource` command has at least [one known
-  bug](https://github.com/Azure/azure-service-operator/issues/3805) requiring the YAML it generates to be
-  edited before it can be applied to a cluster.
```

Comment:
This bug has been fixed: https://github.com/Azure/azure-service-operator/issues/3805